src/Illuminate/Queue/MaxAttemptsExceededException.php

```diff
@@ -6,5 +6,21 @@
 
 class MaxAttemptsExceededException extends RuntimeException
 {
-    //
+    /**
+     * @var \Illuminate\Contracts\Queue\Job
```

Comment:
Could also be `null` when third parties are using this directly without the `forJob` method.

```suggestion
     * @var \Illuminate\Contracts\Queue\Job|null
```